Asterisk.2013/Asterisk.NET/Helper.cs

```diff
@@ -754,6 +754,10 @@ internal static ManagerResponse BuildResponse(Dictionary<string, string> attribu
             int hash, hashEvent;
 
             eventType = attributes["event"].ToLower(CultureInfo);
+            if (eventType == "queuememberpause") //temporary fix for Asterisk 13 pause QueueMemberPause

```

Comment:
Ok, sure.
Well the event name `QueueMemberPause` was added in v12 and has the same properties as `QueueMemberPaused`.
In v13 `QueueMemberPaused` has been removed and `QueueMemberPause` got 2 more parameters `InCall` and `PausedReason` (a specialization class needs to be added for this.):
[Asterisk 13: QueueMemberPause](https://wiki.asterisk.org/wiki/display/AST/Asterisk+13+ManagerEvent_QueueMemberPause)
_I needed the support urgently_